src/components/Button/button.ts

```diff
@@ -0,0 +1,25 @@
+import Block from 'core/Block';
+
+import './button.scss';
+
+interface ButtonProps {
+  className?: string;
```

Comment:
предлагаю рассмотреть вариант с использованием массива строк, вместо одной строки. Так будет легче обрабатывать это свойство, если потребуется
```suggestion
  classes?: string[];
```